### PR TITLE
Add in fiscal year fetch handler

### PR DESCRIPTION
**Pull request by npavlov**

### CODE REVIEW

**Comment by BetimBeja** [comment]
@npavlov, I see green now.
All that is left now is for @jordimontana82 to mege 😃

**Comment by npavlov** [comment]
> @npavlov, I see green now.
> All that is left now is for @jordimontana82 to mege 😃

Yeah, great, thank you for the fix. 😀

**Comment by jordimontana82** [comment]
Hi lads,

Thanks a mill for this, since this is the first "fiscal" date time operator we're introducing, it would be good to have them work the exact same way and possibly looking at a set of properties that define how do you handle the fiscal year (I saw in the PR it's hardcoded to start from april to april, but this really varies between countries and even for the same country is up to each business decide what is going to be their fiscal year (which is normally the calendar year here in Spain for instance).

So I think if we introduce a FiscalYearSettings class where we essentialy we could setup the fiscal year settings (Start Date and Period) at the very least, then all these operators could use it and be consistent and configurable for each customer. 

Thoughts?

More info [here](https://docs.microsoft.com/en-us/dynamics365/customer-engagement/admin/work-fiscal-year-settings)

imagen

**Comment by jordimontana82** [comment]
I would maybe add a public property to the XrmFakedContext class of the same name (FiscalYearSettings) and make these operators consume that.

File: FakeXrmEasy.Shared/FiscalYearSettings.cs [hunk 1]
```diff
@@ -0,0 +1,25 @@
+﻿using Microsoft.Xrm.Sdk;
+using Microsoft.Xrm.Sdk.Client;
+using System;
+
+namespace FakeXrmEasy
+{
+    [EntityLogicalName("organization")]
```
Comment (jordimontana82) [inline FakeXrmEasy.Shared/FiscalYearSettings.cs]: Thanks a mill for this @npavlov ! Curious to know where this EntityLogicalName was generated from...  crmsvcutil or XrmToolkit or similar?

**Comment by jordimontana82** [comment]
That's approved now @npavlov . If you could please reply to the review comment would be nice.

Thanks!

**Comment by jordimontana82** [comment]
@npavlov also, there are other fiscal year operators. Now that that we have the implementation for InFiscalYear operator we could reuse that for the remaining fiscal year operators. I'm going to add it as a separate enhancement issue.

**Comment by jordimontana82** [comment]
Raised #476  for the remaining ones  @npavlov @BetimBeja

**Pull request merged**